lib/dbutils.js

```diff
@@ -118,18 +126,19 @@ dbu.makeValidKey = function makeValidKey (key, length) {
 /**
  * Derive a valid keyspace name from a random bucket name. Try to use valid
  * chars from the requested name as far as possible, but fall back to a sha1
- * if not possible. Also respect Cassandra's limit of 48 or fewer alphanum
+ * if not possible. Also respect SQLite's limit of 48 or fewer alphanum
  * chars & first char being an alpha char.
  *
- * @param {string} reverseDomain, a domain in reverse dot notation
- * @param {string} key, the bucket name to derive the key of
- * @return {string} Valid Cassandra keyspace key
+ * @param {string} domain in dot notation
+ * @param {string} table, the logical table name
+ * @return {string} Valid SQLite keyspace key
  */
-dbu.keyspaceName = function keyspaceName (reverseDomain, key) {
-    var prefix = dbu.makeValidKey(reverseDomain, Math.max(26, 48 - key.length - 3));
+dbu.keyspaceName = function keyspaceName (domain, table) {
```

Comment:
- keyspace does not make sense in sqlite
- use the domain + table name as the database name?